Doc/library/functools.rst

```diff
@@ -492,6 +492,25 @@ The :mod:`functools` module defines the following functions:
      ...     print(arg.real, arg.imag)
      ...
 
+   For code that dispatches on a collections type, e.g. ``list``, but wants
```

Comment:
This might be a personal preference but having the examples in parentheses as opposed to surrounded by commas might help with the readability a bit here and below.

Reply:
I don't have a strong opinion. You have raised my curiosity on what's grammatically correct.

There's some additional rules about surrounding with `,` and independent clauses (whatever those are!). I'm fine with changing it.